src/guake/gconfhandler.py

```diff
@@ -364,7 +364,6 @@ def reload_globals(self, client, connection_id, entry, data):
         method. If more global hotkeys should be added, just connect
         the gconf key to the watch system and add.
         """
-        self.guake.hotkeys.unbind_all()
```

Comment:
Note: there is a method keybinder_unbind_all (https://github.com/engla/keybinder/blob/master/libkeybinder/bind.c)